geode-docs/tools_modules/gfsh/command-pages/create.html.md.erb

```diff
@@ -960,6 +961,48 @@ See [Region Data Storage and Distribution](../../../developing/region_options/ch
 <td><span class="keyword parmname">\-\-partition-resolver</span></td>
 <td>Specifies the full path to a custom partition resolver. To use the provided default partition resolver, specify <code class="ph codeph">org.apache.geode.cache.util.StringPrefixPartitionResolver</code>.</td>
 <td></td>
+</tr>
+<tr>
+<td><span class="keyword parmname">\-\-eviction-enabled</span></td>
+<td>Enables eviction, where the eviction policy is controlled by the resource manager based on heap percentage.</td>
+<td></td>
+</tr>
+<tr>
+<td><span class="keyword parmname">\-\-eviction-entry-count</span></td>
+<td>Enables eviction, where the eviction policy is based on the number of entries in the region.</td>
+<td></td>
+</tr>
+<tr>
+<td><span class="keyword parmname">\-\-eviction-max-memory</span></td>
+<td>Enables eviction, where the eviction policy is based on the amount of memory consumed, specified in megabytes.</td>
+<td></td>
+</tr>
+<tr>
+<td><span class="keyword parmname">\-\-eviction-action</span></td>
+<td>Action to take when the eviction threshold is reached.
+<div class="p">
+<table>
+<tbody>
+<tr class="odd">
+<td>local-destroy</td>
+<td>Entry is destroyed locally. Not available for replicated regions.</td>
+</tr>
+<tr class="even">
+<td>overflow-to-disk</td>
+<td>Entry is overflowed to disk and the value set to null in memory. For partitioned regions, this provides the most reliable read behavior across the region.</td>
```

Comment:
Is it necessary to say `and the value set to null in memory`? Sounds scary and I think it might only be relevant as an internal detail.